.github/workflows/issue-moderator.yml

```diff
@@ -11,7 +11,11 @@ jobs:
   issue-moderator:
     runs-on: ubuntu-latest
```

Comment:
Ubuntu statement from build section applies here.

Reply:
will be done after this pull request as minegame will be reworking the publishing process